app/src/main/res/layout/chapter_filter_popup.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:background="?attr/primaryBlackBackground">
+
+    <!-- Tab Layout -->
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/filter_tabs"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:background="?attr/primaryBlackBackground"
+        app:tabGravity="fill"
+        app:tabMode="fixed"
+        app:tabIndicatorColor="?attr/colorAccent"
+        app:tabSelectedTextColor="?attr/colorAccent"
+        app:tabTextColor="?attr/grayTextColor"
+        app:tabIndicatorHeight="2dp"
+        app:tabTextAppearance="@style/TabNoCaps"
+        app:tabBackground="@drawable/tab_background_dark" />
+
+    <!-- Content Frame -->
+    <FrameLayout
+        android:id="@+id/tab_content"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="16dp">
+
+        <!-- Filter Tab Content -->
+        <LinearLayout
+            android:id="@+id/filter_content"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:visibility="visible">
+
+            <CheckBox
```

Comment:
These check-boxes are in the wrong style, see the style used in the reader settings.